server/pkg/nlslayer/group.go

```diff
@@ -0,0 +1,105 @@
+package nlslayer
+
+import (
+	pl"github.com/reearth/reearth/server/pkg/layer"
+)
+
+type NLSLayerGroup struct {
+	layerBase
+	children            *IDList
+	common				*LayerID
```

Comment:
since `layerBase` contains `Id` why do we need this?

Reply:
This `LayerID` is for common layer https://github.com/reearth/reearth/blob/434b93671b469dbba5c40632656005a150307b42/server/gql/newlayer.graphql#L22